styles/style.css

```diff
@@ -1402,6 +1402,15 @@ ul.tree ul li:last-child:before {
     background-color: yellow !important;
 }
 
+.tei-has-koronasertifikat {
```

Comment:
Forgot the KS prefix?

Reply:
Followed the previous style namings in same context. But agree that ks is better. Will add on all!